### PR TITLE
[v6.2] Fix descriptions for labels and diag-addr parameters for Teleport

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

**Comment by russjones** [comment]
@stevenGravy Is this a backport and was the original PR approved? Do you mind linking?

**Comment by stevenGravy** [comment]
> @stevenGravy Is this a backport and was the original PR approved? Do you mind linking?

Sure, this is a back port of https://github.com/gravitational/teleport/pull/5762  @gus and @awly had approved that one and was waiting on your approval there.

**Review by klizhentas — APPROVED** [review]

**Pull request merged**
